frontend/coprs_frontend/coprs/helpers.py

```diff
@@ -933,3 +933,25 @@ def generate_repo_id_and_name_ext(dependent, url, dep_idx):
         generate_repo_name(url),
     )
     return repo_id, name
+
+
+def multiple_get(dictionary: dict, *keys) -> list:
```

Comment:
Yes, the `payload_multiple_get` will be needed for getting values from payload which is not needed for projects namespace... but  it will be needed in other namespaces migrations so I'll delete it from this PR and introduce in other migration once this is needed - (the same for the rest of "dead" code - I forgot to remove it in the process)